scripts/mdlint.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
is this duplicated with https://github.com/pingcap/docs/blob/master/.markdownlint.yaml file?

Reply:
oh, that's great!

Reply:
can this file be removed?